### DISCUSSION TITLE
ERC20 Deal?

### DISCUSSION THREAD

**Discussion by 0xdeployer** [foundry-rs/foundry#8438] [Q&A] [answered]
Is it possible to add erc20 deal? what would this require? i'll typically find an address with a balance, prank the address, and transfer the token while sending, but if there was an erc20 deal function that would be great.

**Comment by chaoticpunk** [comment, marked as answer]
this?
```sol
function deal(address token, address to, uint256 give) public;
deal(address(dai), alice, 10000e18);
```
